Support/Testing/CircleCI/Dockerfile

```diff
@@ -52,6 +81,7 @@ RUN apt-get install -y flex bison
 # Install x86 compilers
 RUN apt-get install -y g++-multilib
 RUN apt-get install -y clang-6.0
+RUN ln -s clang-6.0 /usr/bin/clang
```

Comment:
Is `/usr/local/bin` in `$PATH` by default? If it is, it might be better to create these symlinks there to avoid potential conflicts in `/usr/bin` in the future. Also, generally speaking, `/usr/bin` is managed by the distribution's package manager and `/usr/local` by the user.

Reply:
That is true, I'll change the paths.